au3/libraries/lib-theme-resources/dark/Components/Colors.txt

```diff
@@ -80,7 +80,7 @@
                     SliderMain: #484f57;
                     SliderDark: #484f57;
                TrackBackground: #23272b;
-                   GraphLabels: #000000;
+                   GraphLabels: #dddee4; // changed from #000000 to #dddee4
```

Comment:
There is no need to add comments explaining your changes. Such a detailed log will bloat the size of the code base.

Besides, developers can also use `git blame` and/or checkout prior versions of the file using `git checkout <ref_hash>`
```suggestion
                   GraphLabels: #dddee4;
```